src/v/cloud_storage/remote_partition.cc

```diff
@@ -455,7 +455,10 @@ class partition_record_batch_reader_impl final
           "{}",
           _reader->config().start_offset,
           _reader->max_rp_offset());
-        if (_reader->is_eof()) {
+        if (
+          _reader->is_eof()
```

Comment:
I think adding the commit comment above the if could be useful, but just a nit. good find.

Reply:
Added a comment, but this condition might change a bit. Will update the comment accordingly.